backend/clubs/admin.py

```diff
@@ -153,6 +153,7 @@ class ClubAdmin(simple_history.admin.SimpleHistoryAdmin):
         "size",
         "application_required",
         "accepting_members",
+        "address",
```

Comment:
I'm not sure if this is too appropriate, since in most cases the addresses will be completely different. This makes filtering by address difficult (you'll have too many items). I would recommend removing this for now.